Makefile

```diff
@@ -261,8 +261,11 @@ examples:
 # api machinery regenerate
 
 apimachinery:
-	#go install ./cmd/libs/go2idl/conversion-gen
+	# conversion: go install ./cmd/libs/go2idl/conversion-gen
 	${GOPATH}/bin/conversion-gen --skip-unsafe=true --input-dirs k8s.io/kops/pkg/apis/kops/v1alpha1 --v=8  --output-file-base=zz_generated.conversion
+	# defaulters: go install ./cmd/libs/go2idl/defaulter-gen
+	${GOPATH}/bin/defaulter-gen --input-dirs k8s.io/kops/pkg/apis/kops/v1alpha1 --v=8  --output-file-base=zz_generated.defaults
+	${GOPATH}/bin/defaulter-gen --input-dirs k8s.io/kops/pkg/apis/kops/v1alpha2 --v=8  --output-file-base=zz_generated.defaults
```

Comment:
How does this target work? And why do we have 2 commands generating the same thing here?